csrankings.csv

```diff
@@ -7930,6 +7930,7 @@ Marta L. Queiros Mattoso,UFRJ,http://cos.ufrj.br/~marta,ZPczFBAAAAAJ
 Marta M. Betcke,University College London,http://www0.cs.ucl.ac.uk/people/M.Betcke.html,HJ3z-24AAAAJ
 Marta Mattoso,UFRJ,http://cos.ufrj.br/~marta,ZPczFBAAAAAJ
 Marta Mejail,University of Buenos Aires,https://www.dc.uba.ar/rrhh/profesores/mejail,WMy2FhwAAAAJ
+Marten van Dijk,University of Connecticut,https://scl.engr.uconn.edu/,byCWPiwAAAAJ

```

Comment:
Not listed here: https://www.cse.uconn.edu/people/faculty/
Please read over inclusion criteria.

Reply:
Including appointment letter from department.  Primary affiliation is ECE but can/does supervise PhD students in CSE.



![martenappointment](https://user-images.githubusercontent.com/17732041/52289153-856f8b80-293b-11e9-8cee-ffe78c5365a7.jpg)